docs/Using-Fleet/Audit-Activities.md

```diff
@@ -583,6 +583,21 @@ This activity contains the following fields:
 }
 ```
 
+### Type `read_host_disk_encryption_key`
+
+Generated when a user reads the disk encryption key for a host.
+
+This activity contains the following fields:
+- "host_display_name": Display name of the host.
```

Comment:
I think those docs need re-generating to update with the host id.

Reply:
thank you <3 as time permits I want to add a CI check for this and also for go.mod, and there's another one that I can't remember now but always annoys me.

Reply:
schema.sql maybe? I see it pop up with unmerged changes every once in a while.